src/lightkurve/io/kbonus.py

```diff
@@ -0,0 +1,105 @@
+"""Reader function for KBONUS-BKG community light curve products."""
+from ..lightcurve import KeplerLightCurve
+from ..collections import LightCurveCollection
+from .generic import read_generic_lightcurve
+from astropy.io import fits
+import numpy as np
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def _warn_bonus_file(filename):
+    with fits.open(filename, lazy_load_hdus=True) as hdulist:
+        if hdulist[0].header["KEPMAG"] < 12:
```

Comment:
this will crash when "KEPMAG" is empty for sources not in the KIC. We should also compare to GMAG

Reply:
great, is GMAG always set?

Reply:
Yes, GMAG always has values.